### DISCUSSION TITLE
Idiomatic way of adding a startup system in 0.10?

### DISCUSSION THREAD

**Discussion by keithsharp** [bevyengine/bevy#8043] [Q&A] [answered]
I've seen both of these patterns used in the documentation and examples:
```rust
App::new()
    .add_startup_system(add_people_system)
    .add_system(greet_people_system)
    .run()
```
or
```rust
App::new()
    .add_systems((
        add_people_system.on_startup(),
        greet_people_system,
    ))
    .run();
```
and was wondering which I should be preferring, and why?

**Comment by NiklasEi** [comment, marked as answer]
This is very much connected to the discussion in #8005 
You can find some pro/contra arguments in there.

It's not yet clear what the API should look like to add systems to schedules (startup systems get collected in a startup schedule). I personally prefere your second example, but you can currently use both approaches.
